SPECS/coreutils/coreutils.spec

```diff
@@ -25,6 +25,8 @@ Patch6:         skip_test_if_run_as_root.patch
 BuildRequires:  libselinux-devel
 Requires:       gmp
 Requires:       libselinux
+BuildRequires:  libselinux-devel
```

Comment:
This looks like a duplicate that's already listed above

Reply:
Fixed.